packages/now-cli/src/commands/deploy/latest.js

```diff
@@ -333,6 +334,7 @@ export default async function main(
     } else {
```

Comment:
Do you need to set `project.sourceFilesOutsideRootDirectory = true` when the project is created? Or is that handled in the API?

Reply:
Ignore that last comment